libs/tup-components/src/system_monitor/details/SystemDetails.module.css

```diff
@@ -0,0 +1,25 @@
+.listing-section > div {
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+.root td {
+    vertical-align: middle;
+  }
```

Comment:
**Leftover Code?** This seems unused, since `styles.root` was deleted from the `SystemDetails.tsx` markup.